src/pages/Swap/index.tsx

```diff
@@ -625,15 +658,27 @@ export default function Swap() {
                     </ButtonLight>
                   </TraceEvent>
                 ) : showWrap ? (
-                  <ButtonPrimary disabled={Boolean(wrapInputError)} onClick={onWrap}>
-                    {wrapInputError ? (
-                      <WrapErrorText wrapInputError={wrapInputError} />
-                    ) : wrapType === WrapType.WRAP ? (
-                      <Trans>Wrap</Trans>
-                    ) : wrapType === WrapType.UNWRAP ? (
-                      <Trans>Unwrap</Trans>
-                    ) : null}
-                  </ButtonPrimary>
+                  <TraceEvent
+                    events={[Event.onClick]}
+                    name={EventName.WRAP_TOKEN_TXN_SUBMITTED}
+                    element={ElementName.WRAP_TOKEN_BUTTON}
+                    properties={formatWrapTokenTxnSubmittedEventProperties(
+                      currencies[Field.INPUT],
+                      currencies[Field.OUTPUT],
+                      parsedAmount
+                    )}
+                    shouldLogImpression={!Boolean(wrapInputError)}
```

Comment:
can you check `!error`?

Reply:
no, this is just the inverse of whats inside disabled={} on line 664.